### PR TITLE
[MGDOBR-348] Setup run local script for integration tests

### PR DESCRIPTION
**Pull request by radtriste**
[MGDOBR-348](https://issues.redhat.com/browse/MGDOBR-348) 

Issue splitted into different PRs for better review:
- https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/364
- https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/365
- https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/366
- https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/367
- https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/368

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

* <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

* <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]
Please mention `run-local-tests.sh` in the test readme .

**Comment by radtriste** [comment]
@sutaakar readme updated

**Comment by sutaakar** [comment]
Thanks

File: dev/bin/manager-run.sh [hunk 1]
```diff
@@ -26,6 +26,10 @@ export KAFKA_CLIENT_SECRET=${ops_client_secret}
 export MANAGED_CONNECTORS_KAFKA_CLIENT_ID=${mc_client_id}
 export MANAGED_CONNECTORS_KAFKA_CLIENT_SECRET=${mc_client_secret}
 
+rm -rf ${LOCAL_ENV_FILE}
+echo "MANAGER_URL=http://localhost:8080" >> ${LOCAL_ENV_FILE}
```
Comment (jomarko) [inline dev/bin/manager-run.sh:30]: is that url correct? in QE internal discussion we mentioned it should be `minikube ip`/manager

**Review by Srihari1192 — APPROVED** [review]
LGTM

**Review by jiripetrlik — COMMENTED** [review]
Looks good to me I was able to run following two scripts successfully:
startMinikubeDeployLocalDev.sh
run-local-tests.sh

I have one comment about dependency management which seems to me quite out of topic of this PR.

File: build-parent/pom.xml [hunk 2]
```diff
@@ -178,6 +178,25 @@
         <version>${version.io.github.classgraph}</version>
         <scope>test</scope>
       </dependency>
+
+      <dependency>
```
[the remark this answers is not in the document]
Comment (radtriste) [reply build-parent/pom.xml:182]: after discussion, we can keep it as it is

**Review by jiripetrlik — APPROVED** [review]

**Pull request merged**